src/main/java/com/sparrowwallet/sparrow/control/CodexKeystoreImportPane.java

```diff
@@ -0,0 +1,256 @@
+        Label shareLabel = new Label("Secret share:");
+        TextField shareField = new TextField();
+        shareField.setPrefColumnCount(27);
+        shareField.textProperty().addListener((observable, oldValue, newValue) -> {
+            secretShareProperty.set(newValue);
+        });
+        shareEntry.getChildren().addAll(shareLabel, shareField);
+        vBox.getChildren().add(shareEntry);
+
+        AnchorPane buttonPane = new AnchorPane();
+        buttonPane.setPadding(new Insets(0, 26, 0, 10));
+
+        validLabel = new Label("Valid checksum", GlyphUtils.getSuccessGlyph());
+        validLabel.setContentDisplay(ContentDisplay.LEFT);
+        validLabel.setGraphicTextGap(5.0);
+        validLabel.managedProperty().bind(validLabel.visibleProperty());
+        validLabel.setVisible(false);
+        buttonPane.getChildren().add(validLabel);
+        AnchorPane.setTopAnchor(validLabel, 5.0);
+        AnchorPane.setLeftAnchor(validLabel, 0.0);
+
+        invalidLabel = new Label("Invalid checksum", GlyphUtils.getInvalidGlyph());
+        invalidLabel.setContentDisplay(ContentDisplay.LEFT);
+        invalidLabel.setGraphicTextGap(5.0);
+        invalidLabel.managedProperty().bind(invalidLabel.visibleProperty());
+        invalidLabel.setVisible(false);
+        buttonPane.getChildren().add(invalidLabel);
+        AnchorPane.setTopAnchor(invalidLabel, 5.0);
+        AnchorPane.setLeftAnchor(invalidLabel, 0.0);
+
+        secretShareProperty.addListener((ChangeListener<String>) (c, oldval, newval) -> {
+            boolean empty = secretShareProperty.isEmpty().get();
+            boolean validChecksum = false;
+            onInputChange(empty, validChecksum);
+        });
+
+        HBox rightBox = new HBox();
+        rightBox.setSpacing(10);
+
+        calculateButton = new Button("Create Keystore");
+        calculateButton.setDisable(true);
+        calculateButton.setDefaultButton(!AppServices.onlineProperty().get());
+        calculateButton.managedProperty().bind(calculateButton.visibleProperty());
+        calculateButton.setTooltip(new Tooltip("Create the keystore from the provided secret share"));
+        calculateButton.setOnAction(event -> {
+            setExpanded(true);
+            enterCodexButton.setVisible(false);
+            importButton.setVisible(true);
+            importButton.setDisable(false);
+            setDescription("Ready to import");
+            showHideLink.setText("Show Derivation...");
+            showHideLink.setVisible(false);
+            setContent(getDerivationEntry(getDefaultDerivation()));
+        });
+
+        rightBox.getChildren().add(calculateButton);
+
+        buttonPane.getChildren().add(rightBox);
+        AnchorPane.setRightAnchor(rightBox, 0.0);
+
+        vBox.getChildren().add(buttonPane);
+
+        Platform.runLater(shareField::requestFocus);
+
+        StackPane stackPane = new StackPane();
+        stackPane.getChildren().add(vBox);
+        return stackPane;
```

Comment:
Replace with:
```
return vBox;
```

The `StackPane` is unnecessary and introduces unwanted layout changes.